### ISSUE TITLE
Process not die after closing UI

### ISSUE THREAD

**Issue by bit0rez** [DockStation/dockstation#36]
Hi!
I found some bug. When i close program windows, process not died. I can run many dockstation processes and after closing programs they still alive.

I use DockStation v1.2.0 on Ubuntu 16.04 LTS. I can reproduse this bug at DockStation v1.1.0.

P.S.: Sorry for my bad english. :)

**Comment by igor-lemon** [comment]
Hi @bit0rez. What is process name? Because after starting runs few processes.

**Comment by bit0rez** [comment]
After i run dockstation from DM (ALT+F2) and closed main window.
```
 ps ax | grep dockstation
22805 ?        Sl     0:04 /opt/DockStation/dockstation
22807 ?        S      0:00 /opt/DockStation/dockstation --type=zygote --no-sandbox
22835 ?        Sl     0:00 /opt/DockStation/dockstation --type=gpu-process --no-sandbox --supports-dual-gpus=false --gpu-driver-bug-workarounds=7,23,74 --disable-gl-extensions=GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent --gpu-vendor-id=0x10de --gpu-device-id=0x1201 --gpu-driver-vendor --gpu-driver-version --gpu-driver-date --service-request-channel-token=C1389D85FF36355D4ACD1945BA00DE74
22841 ?        Sl     0:04 /opt/DockStation/dockstation --type=renderer --no-sandbox --primordial-pipe-token=1F55E748D56E166D4FD2A8A5C38F9355 --lang=ru --app-path=/opt/DockStation/resources/app.asar --node-integration=true --webview-tag=true --background-color=#ffffff --enable-pinch --num-raster-threads=2 --enable-main-frame-before-activation --content-image-texture-target=0,0,3553;0,1,3553;0,2,3553;0,3,3553;0,4,3553;0,5,3553;0,6,3553;0,7,3553;0,8,3553;0,9,3553;0,10,3553;0,11,3553;0,12,3553;0,13,3553;0,14,3553;0,15,3553;1,0,3553;1,1,3553;1,2,3553;1,3,3553;1,4,3553;1,5,3553;1,6,3553;1,7,3553;1,8,3553;1,9,3553;1,10,3553;1,11,3553;1,12,3553;1,13,3553;1,14,3553;1,15,3553;2,0,3553;2,1,3553;2,2,3553;2,3,3553;2,4,3553;2,5,3553;2,6,3553;2,7,3553;2,8,3553;2,9,3553;2,10,3553;2,11,3553;2,12,3553;2,13,3553;2,14,3553;2,15,3553;3,0,3553;3,1,3553;3,2,3553;3,3,3553;3,4,3553;3,5,3553;3,6,3553;3,7,3553;3,8,3553;3,9,3553;3,10,3553;3,11,3553;3,12,3553;3,13,3553;3,14,3553;3,15,3553;4,0,3553;4,1,3553;4,2,3553;4,3,3553;4,4,3553;4,5,3553;4,6,3553;4,7,3553;4,8,3553;4,9,3553;4,10,3553;4,11,3553;4,12,3553;4,13,3553;4,14,3553;4,15,3553 --disable-accelerated-video-decode --service-request-channel-token=1F55E748D56E166D4FD2A8A5C38F9355 --renderer-client-id=4 --shared-files=v8_natives_data:100,v8_snapshot_data:101
23379 pts/8    S+     0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn dockstatio
```
If i run dockstation in terminal and close main window, terminal stay busy.
```
$ dockstation 
[]
[]

```
I terminate process by press Ctrl-C, then i run dockstation from DM and close main window.
```
ps ax | grep dockstation
22805 ?        Sl     0:04 /opt/DockStation/dockstation
22807 ?        S      0:00 /opt/DockStation/dockstation --type=zygote --no-sandbox
22835 ?        Sl     0:00 /opt/DockStation/dockstation --type=gpu-process --no-sandbox --supports-dual-gpus=false --gpu-driver-bug-workarounds=7,23,74 --disable-gl-extensions=GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent --gpu-vendor-id=0x10de --gpu-device-id=0x1201 --gpu-driver-vendor --gpu-driver-version --gpu-driver-date --service-request-channel-token=C1389D85FF36355D4ACD1945BA00DE74
22841 ?        Sl     0:04 /opt/DockStation/dockstation --type=renderer --no-sandbox --primordial-pipe-token=1F55E748D56E166D4FD2A8A5C38F9355 --lang=ru --app-path=/opt/DockStation/resources/app.asar --node-integration=true --webview-tag=true --background-color=#ffffff --enable-pinch --num-raster-threads=2 --enable-main-frame-before-activation --content-image-texture-target=0,0,3553;0,1,3553;0,2,3553;0,3,3553;0,4,3553;0,5,3553;0,6,3553;0,7,3553;0,8,3553;0,9,3553;0,10,3553;0,11,3553;0,12,3553;0,13,3553;0,14,3553;0,15,3553;1,0,3553;1,1,3553;1,2,3553;1,3,3553;1,4,3553;1,5,3553;1,6,3553;1,7,3553;1,8,3553;1,9,3553;1,10,3553;1,11,3553;1,12,3553;1,13,3553;1,14,3553;1,15,3553;2,0,3553;2,1,3553;2,2,3553;2,3,3553;2,4,3553;2,5,3553;2,6,3553;2,7,3553;2,8,3553;2,9,3553;2,10,3553;2,11,3553;2,12,3553;2,13,3553;2,14,3553;2,15,3553;3,0,3553;3,1,3553;3,2,3553;3,3,3553;3,4,3553;3,5,3553;3,6,3553;3,7,3553;3,8,3553;3,9,3553;3,10,3553;3,11,3553;3,12,3553;3,13,3553;3,14,3553;3,15,3553;4,0,3553;4,1,3553;4,2,3553;4,3,3553;4,4,3553;4,5,3553;4,6,3553;4,7,3553;4,8,3553;4,9,3553;4,10,3553;4,11,3553;4,12,3553;4,13,3553;4,14,3553;4,15,3553 --disable-accelerated-video-decode --service-request-channel-token=1F55E748D56E166D4FD2A8A5C38F9355 --renderer-client-id=4 --shared-files=v8_natives_data:100,v8_snapshot_data:101
23638 ?        Sl     0:01 /opt/DockStation/dockstation
23640 ?        S      0:00 /opt/DockStation/dockstation --type=zygote --no-sandbox
23665 ?        Sl     0:00 /opt/DockStation/dockstation --type=gpu-process --no-sandbox --supports-dual-gpus=false --gpu-driver-bug-workarounds=7,23,74 --disable-gl-extensions=GL_KHR_blend_equation_advanced GL_KHR_blend_equation_advanced_coherent --gpu-vendor-id=0x10de --gpu-device-id=0x1201 --gpu-driver-vendor --gpu-driver-version --gpu-driver-date --service-request-channel-token=F7DE0A1FBC6A0AF163F265A857F57802
23672 ?        Sl     0:02 /opt/DockStation/dockstation --type=renderer --no-sandbox --primordial-pipe-token=358D9FEB3735148A801C7D52DCF05976 --lang=ru --app-path=/opt/DockStation/resources/app.asar --node-integration=true --webview-tag=true --background-color=#ffffff --enable-pinch --num-raster-threads=2 --enable-main-frame-before-activation --content-image-texture-target=0,0,3553;0,1,3553;0,2,3553;0,3,3553;0,4,3553;0,5,3553;0,6,3553;0,7,3553;0,8,3553;0,9,3553;0,10,3553;0,11,3553;0,12,3553;0,13,3553;0,14,3553;0,15,3553;1,0,3553;1,1,3553;1,2,3553;1,3,3553;1,4,3553;1,5,3553;1,6,3553;1,7,3553;1,8,3553;1,9,3553;1,10,3553;1,11,3553;1,12,3553;1,13,3553;1,14,3553;1,15,3553;2,0,3553;2,1,3553;2,2,3553;2,3,3553;2,4,3553;2,5,3553;2,6,3553;2,7,3553;2,8,3553;2,9,3553;2,10,3553;2,11,3553;2,12,3553;2,13,3553;2,14,3553;2,15,3553;3,0,3553;3,1,3553;3,2,3553;3,3,3553;3,4,3553;3,5,3553;3,6,3553;3,7,3553;3,8,3553;3,9,3553;3,10,3553;3,11,3553;3,12,3553;3,13,3553;3,14,3553;3,15,3553;4,0,3553;4,1,3553;4,2,3553;4,3,3553;4,4,3553;4,5,3553;4,6,3553;4,7,3553;4,8,3553;4,9,3553;4,10,3553;4,11,3553;4,12,3553;4,13,3553;4,14,3553;4,15,3553 --disable-accelerated-video-decode --service-request-channel-token=358D9FEB3735148A801C7D52DCF05976 --renderer-client-id=4 --shared-files=v8_natives_data:100,v8_snapshot_data:101
23771 pts/8    S+     0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn dockstation
```
Old processes not died and new was run.

**Comment by bit0rez** [comment]
After command `killall dockstation`
```
➜  ~ killall dockstation
➜  ~ ps ax | grep dockstation
22805 ?        Sl     0:04 /opt/DockStation/dockstation
22807 ?        Z      0:00 [dockstation] <defunct>
23638 ?        Sl     0:01 /opt/DockStation/dockstation
23640 ?        Z      0:00 [dockstation] <defunct>
24077 pts/8    S+     0:00 grep --color=auto --exclude-dir=.bzr --exclude-dir=CVS --exclude-dir=.git --exclude-dir=.hg --exclude-dir=.svn dockstation
```
All zombies was killed by second run of this command.

**Comment by igor-lemon** [comment]
@bit0rez fixed v1.2.1. Please check you too.

**Comment by bit0rez** [comment]
Fixed! Thx! )
